src/main/resources/PterodactylPowerAction/Bundle_en.properties

```diff
@@ -13,5 +13,17 @@ command.reload.success=Configuration successfully reloaded!
 command.reload.error=Configuration contains error and could not be reloaded. Check the server's console for details.
 command.doctor.start=Performing diagnostic checks... Check the server's console for details.
 command.clear.start=Stopping empty servers...
+bossbar.starting={0} is starting
+bossbar.updating.config=Updating configuration
+bossbar.setting.permissions=Setting file permissions
+bossbar.pulling.image=Pulling container image
+bossbar.starting.jvm=Starting JVM
+bossbar.loading.server=Loading server
+bossbar.loading.libs=Loading libraries
+bossbar.binding.port=Binding to port
+bossbar.loading.plugins=Loading plugins
+bossbar.preparing.world=Preparing world
```

Comment:
I'm not sure I'd like to go into such details in the messages. The players does not have to know how a server starts. I think the bossbar should only display "Server is starting" without more details all the while updating the bossbar's progress.